Sentry.xcodeproj/project.pbxproj

```diff
@@ -3634,8 +3642,6 @@
 				8ECC674325C23A1F000E2BF6 /* SentrySpanContext.m */,
 				622C08D929E554B9002571D4 /* SentrySpanContext+Private.h */,
 				8E4E7C7325DAAB49006AB9E2 /* SentrySpanProtocol.h */,
-				7B3B83712833832B0001FDEB /* SentrySpanOperations.h */,
-				622C08D729E546F4002571D4 /* SentryTraceOrigins.h */,
```

Comment:
last one of these. honestly not even sure how it's building without these, since in this section they were removed instead of just relocated. Maybe they're not actually needed in whatever target this is?

Reply:
Seems to be left-overs, these headers were removed in #4717 and #4718